### ISSUE TITLE
Связать высказывание с событием

### ISSUE THREAD

**Issue by kirkkonen** [kromkrom/NetForSpeech#11]
На странице редактирования или добавления высказывания или события добавить селект со списком, соответственно, событий или высказываний для того, чтобы их связать. Пример: в событиях есть то, как я съездил в Финляндию. В высказываниях есть моя цитата по этому поводу. И их никак не связать!

**Comment by kirill-gr** [comment]
Можно просто связать высказывания и факты (в любых комбинациях), но это достаточно просто. Давай сразу нагрузим связь дополнительной информацией.

**Comment by kirkkonen** [comment]
Согласен. Нужен селект/радиобаттон с выбором, противоречит или подтверждает. Плюс поле для комментария, как-то так наверное.

**Comment by kirill-gr** [comment]
У меня вопрос. А кто будет связывать высказывания и факты? Модераторы или простые пользователи?

**Comment by kirkkonen** [comment]
Я думаю, что пользователям нужно дать такую возможность, а модератор будет проверять и редактировать при необходимости.

**Comment by kirill-gr** [comment]
Теперь при создании фактов и высказываний можно связывать их в любых комбинациях
